wgsl/index.bs

```diff
@@ -2846,6 +2846,9 @@ The element count expression |N| of a fixed-size array is subject to the followi
 
 Note:  The element count value is fully determined at [=pipeline creation=] time.
 
+Note:  To qualify for type-equivalency, the override expression must be an **identifier**.
```

Comment:
I'm probably being super pedantic, but the spec uses "override expression" as a specific term for a larger set of things (e.g. it includes const expressions). With that meaning in mind, this note is arguably inaccurate.

Reply:
```suggestion
Note:  To qualify for type-equivalency, any override expression that is not a const expression must be an **identifier**.
```